tekton/pipeline-release-run-5bgpg.log

```diff
@@ -0,0 +1,10128 @@
+[precheck : git-source-tekton-pipelines-git-v0-10-0-fxv9c] {"level":"info","ts":1579772695.226866,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/refs/heads/master: no such file or directory"}
```

Comment:
Did you mean to include this gigantic file?

Reply:
I did not 🧐😆